NUI.podspec

```diff
@@ -1,12 +1,12 @@
 Pod::Spec.new do |s|
   s.name         = "NUI"
-  s.version      = "0.5.5"
+  s.version      = "0.5.6"
   s.summary      = "Style iOS apps with a stylesheet, similar to CSS."
   s.description  = "NUI is a drop-in UI kit for iOS that lets you style UI elements using a stylesheet, similar to CSS. It lets you style an entire app in minutes."
-  s.homepage     = "https://github.com/tombenner/nui"
+  s.homepage     = "https://github.com/Joulebug/nui"
   s.license      = { :type => 'MIT', :file => 'LICENSE.txt' }
   s.author       = { "Tom Benner" => "tombenner@gmail.com" }
-  s.source       = { :git => "https://github.com/tombenner/nui.git", :tag => "v0.5.5" }
+  s.source       = { :git => "https://github.com/Joulebug/nui.git", :tag => "v0.5.6" }
```

Comment:
If you need an intermediary release of your own fork, please make such changes on a different branch than you're using to file a pull request.